packages/cv-wrapper/src/plugins/i18n.ts

```diff
@@ -0,0 +1,22 @@
+import { createI18n } from 'vue-i18n';
+
+import en from '@/locales/en.json';
+
+export const supportedLocales = ['en'];
+
+export type SupportedLocale = (typeof supportedLocales)[number];
+
+export const defaultLocale: SupportedLocale = 'en';
+
+type MessageSchema = typeof en;
+
+const i18n = createI18n<[MessageSchema], SupportedLocale>({
+  legacy: false,
+  locale: defaultLocale,
```

Comment:
Browser defaults coming later in other issue?

Reply:
Yes :) This issue is just for the initial setup :)